### PR TITLE
feat: update ux with convesation mode layout

### PR DESCRIPTION
**Pull request by unimu-cic**

### CODE REVIEW

File: views/hello-world.jsx [hunk 1]
```diff
@@ -111,155 +74,38 @@ const FormDefaultExample = () => {
...
-                <FormFooter>
-                  <ButtonGroup>
-                    <LoadingButton
-                      type='submit'
-                      appearance='primary'
-                      isLoading={submitting}
-                    >
-                      Submit
-                    </LoadingButton>
-                    <Button type={'button'} appearance={'subtle'} onClick={() => {AP.dialog.close()}}>Close</Button>
```
Comment (MrCoder) [inline views/hello-world.jsx:238]: Close button is needed somewhere on the page.

**Review by MrCoder — CHANGES_REQUESTED** [review]
"Close" button is needed.

File: views/components/MessageSender.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            </Field>
+            <ButtonBox>
+              <Popup
+                isOpen={isOpen}
+                onClose={() => setIsOpen(false)}
+                content={() => <PreDefinedPrompts onSelect={handleSelect}/>}
+                placement="bottom-start"
+                trigger={(triggerProps) => (
+                  <span {...triggerProps} onClick={() => setIsOpen(!isOpen)}>
+                      <LightbulbFilledIcon label='Predefined prompts'/>
```
[the remark this answers is not in the document]
Comment (unimu-cic) [reply views/components/MessageSender.jsx]: Added.

File: views/hello-world.jsx [hunk 3]
```diff
@@ -1,86 +1,49 @@
...
-  const [tokenUsageRatio, setTokenUsageRatio] = React.useState()
-  const [copied, setCopied] = React.useState(false);
-  const copyBtnRef = React.useRef();
-  const inputRef = React.useRef();
-  const hasError = messages.startsWith('An error occurred')
+  const [sessions, setSessions] = React.useState([]);
 
-  let onSubmit = async (data) => {
-    setMessages('')
+  const handleSubmit = React.useCallback(async (prompt) => {
```
[the remark this answers is not in the document]
Comment (unimu-cic) [reply views/hello-world.jsx:47]: > Is there a simple way to support `ENTER` for sending and `SHIFT+ENTER` for new line?

Added

File: views/hello-world.jsx (hunk 1 above)
Comment (unimu-cic) [reply views/hello-world.jsx:238]: Added:

image

File: views/components/MessageSender.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (unimu-cic) [reply views/components/MessageSender.jsx]: > Add some mouse hover effect?

Added:
image

**Review by MrCoder — APPROVED** [review]

**Pull request merged**